client/main/variableeditor.js

```diff
@@ -285,6 +287,13 @@ const EditorWidget = Backbone.View.extend({
                 event.preventDefault();
                 event.stopPropagation();
             }
+            else if (keypressed == 27) {
```

Comment:
can you make this something like:

    else if (keypressed == 27) {  // escape key

or whatever the key is. oh, and it should probably have three equals signs.